aiida/cmdline/commands/cmd_import.py

```diff
@@ -127,7 +127,7 @@ def cmd_import(archives, webpages, group, extras_mode_existing, extras_mode_new,
                 extras_mode_new=extras_mode_new,
                 comment_mode=comment_mode)
         except exceptions.IncompatibleArchiveVersionError as exception:
-            echo.echo_warning('{} cannot be imported: {}'.format(archive, exception))
+            echo.echo_critical('{} cannot be imported: {}'.format(archive, exception))
             continue
```

Comment:
Since the `echo_critical` call will call `sys.exit` you can remove the `continue`

Reply:
Done.